consumer/pact-jvm-consumer/src/test/java/au/com/dius/pact/consumer/PactDslJsonBodyTest.java

```diff
@@ -265,7 +265,7 @@ public void testLargeDateFormat() {
         .date("lastUpdate", DATE_FORMAT)
         .date("creationDate", DATE_FORMAT);
       JSONObject jsonObject = (JSONObject) response.getBody();
-      assertThat(jsonObject.get("lastUpdate").toString(), matchesPattern("\\w{3}\\.?, \\d{2} \\w{3}\\.? \\d{4} \\d{2}:00:00 \\+\\d+ GMT"));
+      assertThat(jsonObject.get("lastUpdate").toString(), matchesPattern("\\w{2,3}\\.?, \\d{2} \\w{3}\\.? \\d{4} \\d{2}:00:00 \\+\\d+ GMT"));
```

Comment:
This test is not related to my PR, but failed when I cloned the repo and ran the tests. The day value was 'Mo' for me, not sure why, maybe because of my locale (de)?
In German, 'Mo' is short for Montag which is Monday. :)

Reply:
Damn i18n, always there to catch you out.